apps/reflect/src/Context/ColorModeContext.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { ReactNode, createContext, useEffect, useState } from "react";
+import { createTheme, ThemeProvider } from "@mui/material/styles";
+import { getDesignTokens, getThemedComponents } from "src/utils/theme";
+
+interface ColorModeContextType {
+  toggleColorMode: () => void;
+  colorMode: "dark" | "light";
+}
+
+export const ColorModeContext = createContext<ColorModeContextType>({
+  toggleColorMode: () => {},
+  colorMode: "dark",
+});
+
+export const ColorModeProvider = ({ children }: { children: ReactNode }) => {
+  const [colorMode, setColorMode] = useState<"light" | "dark">("dark");
+
+  useEffect(() => {
+    const localColorMode = localStorage.getItem("theme");
+    if (localColorMode) {
+      setColorMode(localColorMode as "light" | "dark");
+    }
```

Comment:
In this section, it would be more appropriate to assign a type to the 'localColorMode' value
for instance   `const localColorMode:IlocalColorMode = localStorage.getItem("theme");`